java-frontend/src/main/java/org/sonar/java/resolve/BytecodeAnnotationVisitor.java

```diff
@@ -52,7 +52,16 @@ public AnnotationVisitor visitAnnotation(String name, String desc) {
 
   @Override
   public void visitEnum(String name, String desc, String value) {
-    getSymbol(desc).members().lookup(value).stream()
+    JavaSymbol.TypeJavaSymbol sym = getSymbol(desc);
+    if(sym.completing) {
+      sym.callbackOnceComplete(() -> addSymbolAsValue(name, value, sym));
+      return;
```

Comment:
why not simply an `else` statement here?

Reply:
reminder after refactoring and as style was going I did not feel the need to use a else... not really sure what's best here.